### PR TITLE
Porting shallow Water application to pybind

### PR DESCRIPTION
**Pull request by miguelmaso**
I have applied the changes explained in https://github.com/KratosMultiphysics/Kratos/wiki/Porting-to-PyBind11---common-steps

However I get the next compilation warning:
CMakeFiles/KratosShallowWaterApplication.dir/custom_python/add_custom_utilities_to_python.cpp.o (symbol from plugin): aviso: memset used with constant zero length parameter; this could be due to transposed parameters

### CODE REVIEW

**Comment by miguelmaso** [comment]
#1513

File: applications/ShallowWaterApplication/custom_python/add_custom_strategies_to_python.cpp [hunk 1]
```diff
@@ -52,12 +47,12 @@ namespace Kratos
 			//********************************************************************
 			//********************************************************************
 // 			class_< TestStrategy< SparseSpaceType, LocalSpaceType, LinearSolverType >,
```
Comment (RiccardoRossi) [inline applications/ShallowWaterApplication/custom_python/add_custom_strategies_to_python.cpp:49]: maybe it is a good time to take out the comments :-)

**Review by RiccardoRossi — COMMENTED** [review]
looks good to me. do the tests run?

one question before you merge: this is a PR on the top of the pybind11 branch right?

**Review by RiccardoRossi — APPROVED** [review]
i answered myself, sorry for the noise. i am approving, however please ensure that your own tests pass

**Comment by miguelmaso** [comment]
Thank you
The tests run. I should add some standard tests to the application folder. And also I should make a cleanup

**Pull request merged**